pkgs/top-level/all-packages.nix

```
@@ -3169,6 +3169,8 @@ with pkgs;
 
   ghidra-bin = callPackage ../tools/security/ghidra { };
 
+  gh2md = python3Packages.callPackage ../tools/backup/gh2md { };
```

```suggestion
  gh2md = callPackage ../tools/backup/gh2md { };
```

using callPackage from sub-package sets is discouraged